modules/openy_map_lb/composer.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "drupal/openy_map_lb",
```

Comment:
this is not going to work
composer.json should be in the root directory only @david-storm